restli-int-test/src/test/java/com/linkedin/restli/examples/TestHttp2With204AndException.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.restli.examples;
+
+import com.linkedin.r2.RemoteInvocationException;
+import com.linkedin.r2.transport.http.client.HttpClientFactory;
+import com.linkedin.r2.transport.http.common.HttpProtocolVersion;
+import com.linkedin.restli.client.Request;
+import com.linkedin.restli.client.Response;
+import com.linkedin.restli.common.HttpStatus;
+import com.linkedin.restli.examples.greetings.api.Greeting;
+import com.linkedin.restli.examples.greetings.client.GreetingsBuilders;
+import com.linkedin.restli.server.RestLiConfig;
+import com.linkedin.restli.test.util.RootBuilderWrapper;
+import java.util.HashMap;
+import java.util.Map;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class TestHttp2With204AndException extends RestLiIntegrationTest
+{
+  @BeforeClass
+  public void initClass() throws Exception
+  {
+    Map<String, String> transportProperties = new HashMap<>();
+    final String httpRequestTimeout = System.getProperty("test.httpRequestTimeout", "100000");
+    transportProperties.put(HttpClientFactory.HTTP_REQUEST_TIMEOUT, httpRequestTimeout);
+    transportProperties.put(HttpClientFactory.HTTP_PROTOCOL_VERSION, HttpProtocolVersion.HTTP_2.name());
+    super.init(false, new RestLiConfig(), transportProperties);
+  }
+
+  @AfterClass
+  public void shutDown() throws Exception
+  {
+    super.shutdown();
+  }
+
+  @Test(expectedExceptions = RemoteInvocationException.class)
```

Comment:
You should remove expectedExceptions. What this means is you expect the test to throw the exception and it is a failure if no exception was thrown.

Reply:
+1

Reply:
Removed